doc/cookbook/source/examples/classifier/random_forest.rst

```diff
@@ -0,0 +1,38 @@
+============================
+Random Forest classification
+============================
+
+Random Forest is an ensemble learning method which implements multiple decision trees while training and performs prediction by using a combination rule on the outputs of individual decision trees.
+
+-------
+Example
+-------
+
+CDenseFeatures (here 64 bit floats aka RealFeatures) and :sgclass:`CMulticlassLabels` are created from training and test data file
+
+.. sgexample:: random_forest.sg:create_features
+
+A combination rule of :sgclass:`CMajorityVote` is created to be used as a combination rule. This and other combination rules are derived form the :sgclass:`CCombinationRule` .
```

Comment:
Can you maybe phrase all this in the way that there is a default configuration, and then *optionally* one can put in these decision rules. See what I mean?